internal/db/models.go

```diff
@@ -564,6 +564,11 @@ type ProfilesWithEntityProfile struct {
 	Profid          uuid.UUID             `json:"profid"`
 }
 
+type ProfilesWithSelector struct {
+	Profid    uuid.UUID         `json:"profid"`
+	Selectors []ProfileSelector `json:"selectors"`
+}
```

Comment:
This nice and simple struct is pretty much why I've been doing the dance with the override. I wasn't able to force sqlc to generate the proper type otherwise.